tests/integration/components/frost-fixed-table-test.js

```diff
@@ -327,4 +344,154 @@ describe(test.label, function () {
       })
     })
   })
+
+  describe('events', function () {
+    let dispatcher
+
+    beforeEach(function () {
+      dispatcher = sandbox.stub()
+      this.setProperties({
+        actions: {dispatcher},
+        fixedColumns: fixedColumnsWithCustomRenderers
+      })
+
+      this.render(hbs`
+        {{frost-fixed-table
+          callbackHandler=(action 'dispatcher')
```

Comment:
You don't need to jump through the `actions` object hoop. You can set the property directly and pass that in (I don't even think you need the `action` helper, but it doesn't hurt. In this case, if the property name were `onCallback` as I've suggested, your actually handler stub can be called `callbackHandler`

```js
callbackHandler = sandbox.stub()
this.setProperties({
  callbackhandler,
  ...
})

this.render(hbs`
  {{frost-fixed-table
    onCallback=(action callbackHandler)
    ...
  }}
`)
```
```